.changeset/peaceful-minds-rest.md

```diff
@@ -2,4 +2,4 @@
 "@biomejs/biome": patch
 ---
 
-Fix a panic related to bogus import statements in `useExhaustiveDependencies` ([#4568](https://github.com/biomejs/biome/issues/4568)).
+Fixed [#4568](https://github.com/biomejs/biome/issues/4568): Bogus import statements no longer can cause a panic in `useExhaustiveDependencies`.
```

Comment:
I suggest to avoid the use of "bogus" because it's our internal terminology, borrowed from Roslyn. I suggest the of "broken" or "incorrect"